shell/dialog/AddCustomBadgeDialog.vue

```diff
@@ -43,19 +45,18 @@ export default {
   },
   mounted() {
     // Generates a fake cluster object for use with badge component on cluster provisioning.
-
-    if (this.isCreate) {
+    if (this.enableFields) {
```

Comment:
Does `enableFields` computed really get asserted in the `mounted` lifecycle? 🤔 

Always thought `mounted` might run before the first computed's get fired... For sure it doesn't go back to `mounted` if the computed has changed